apps/climatemappedafrica/src/components/HowItWorks/index.js

```diff
@@ -0,0 +1,200 @@
+import { RichText } from "@commons-ui/payload";
+import { useMediaQuery, Box, Button, Grid, Typography } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Player from "@/climatemappedafrica/components/HowItWorks/Player";
+import useStyles from "@/climatemappedafrica/components/HowItWorks/useStyles";
+import Section from "@/climatemappedafrica/components/Section";
+
+function HowItWorks({
+  title,
+  description,
+  link,
+  video,
+  backgroundImage,
+  foregroundImage,
+  ...props
+}) {
+  const classes = useStyles(props);
+
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("md"));
+
+  return (
+    <Box
+      sx={{
+        height: theme.typography.pxToRem(820),
+        position: "relative",
+        [theme.breakpoints.up("md")]: {
+          height: theme.typography.pxToRem(618),
+        },
+        [theme.breakpoints.up("lg")]: {
+          height: theme.typography.pxToRem(728),
+        },
+      }}
+    >
+      <Box
+        sx={{
+          position: "absolute",
+          height: theme.typography.pxToRem(390),
+          width: "100%",
+          zIndex: -1,
+          [theme.breakpoints.up("md")]: {
+            height: theme.typography.pxToRem(618),
+          },
+          [theme.breakpoints.up("lg")]: {
+            height: theme.typography.pxToRem(728),
+          },
+          backgroundImage: `url(${backgroundImage.src})`,
+          backgroundSize: "cover",
+        }}
+      />
+      <Box
+        sx={{
+          display: {
+            xs: "none",
+            md: "block",
+            lg: "none",
+          },
+        }}
+      >
+        <Box
+          sx={{
+            position: "absolute",
+            left: 0,
+            top: theme.typography.pxToRem(42),
+            width: "100%",
+            background: `linear-gradient(to right, #ffffffE6 0%, #ffffffE6 56%, transparent 56%, transparent 100%)`,
+            height: theme.typography.pxToRem(524),
+          }}
+        />
+      </Box>
+      <Section classes={{ root: classes.section }}>
+        <Grid container direction={isMobile ? "column-reverse" : "row"}>
```

Comment:
You don't need `isMobile` for this
```suggestion
        <Grid container direction={{ xs: "column-reverse", lg: "row" }}>
```